<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialmonitorwidget</class>
 <widget class="QWidget" name="serialmonitorwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="messageTxt"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="outputText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ImageViewer" name="outputImage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="autoScrollCheck">
       <property name="text">
        <string>Autoscroll</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="runSketchCheck">
       <property name="text">
        <string>Run Sketch</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="lineEndingBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>No line ending</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Newline</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Carriage return</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Both NL &amp; CR</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="baudrateBox">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>300 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28800 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600 baud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200 baud</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header>imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
